tests/data_utils.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+import platform
+
+from future.utils import old_div
+
+from quantized_mesh_tile import TerrainTopology
+from quantized_mesh_tile.editable_terrain import EditableTerrainTile
+from quantized_mesh_tile.global_geodetic import GlobalGeodetic
+
+MAX = 32767.0
```

Comment:
You can remove this MAX, it is now part of the terrain tile class. https://github.com/loicgasser/quantized-mesh-tile/blob/master/quantized_mesh_tile/terrain.py#L186

Reply:
ok